### PR TITLE
executor: support bit_and/bit_xor func in new agg evaluation framework

### PR DESCRIPTION
**Pull request by XuHuaiyu**
Thank you for working on TiDB! Please read TiDB's [CONTRIBUTING](https://github.com/pingcap/tidb/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.

## What have you changed? (mandatory)

support bit_and/ bit_xor functions in new agg evaluation framework.

## What are the type of the changes (mandatory)?

The currently defined types are listed below, please pick one of the types for this PR by removing the others:

- Improvement 

## How has this PR been tested (mandatory)?

the existing tests

## Does this PR affect documentation (docs/docs-cn) update? (optional)
no

## Refer to a related PR or issue link (optional)
#6952

### CODE REVIEW

**Comment by XuHuaiyu** [comment]
wait for #7002

**Comment by zz-jason** [comment]
@XuHuaiyu plz fix Ci

**Comment by XuHuaiyu** [comment]
PTAL @zz-jason

File: executor/aggfuncs/func_bitfuncs.go [hunk 1]
```diff
@@ -17,16 +17,13 @@ import (
 	"github.com/juju/errors"
 	"github.com/pingcap/tidb/sessionctx"
 	"github.com/pingcap/tidb/util/chunk"
+	"math"
```
Comment (zz-jason) [inline executor/aggfuncs/func_bitfuncs.go]: move this up and leave an empty line between other packages

**Comment by XuHuaiyu** [comment]
PTAL @winoros @crazycs520

File: executor/aggfuncs/builder.go [hunk 2]
```diff
@@ -135,10 +135,26 @@ func buildBitOr(aggFuncDesc *aggregation.AggFuncDesc, ordinal int) AggFunc {
 
 // buildCount builds the AggFunc implementation for function "BIT_XOR".
 func buildBitXor(aggFuncDesc *aggregation.AggFuncDesc, ordinal int) AggFunc {
+	switch aggFuncDesc.Args[0].GetType().EvalType() {
```
Comment (winoros) [inline executor/aggfuncs/builder.go]: Add todo to tell that we need to implement distinct one?

Comment (XuHuaiyu) [reply executor/aggfuncs/builder.go]: @winoros 
MySQL does not support the syntax `bit_or(distinct), bit_and(distinct), bit_xor(distinct)`

Comment (winoros) [reply executor/aggfuncs/builder.go]: If so, remove this [comment](https://github.com/pingcap/tidb/pull/7004/files#diff-f79c13632bdc165cfe6ed19c1a910d62R124) or add it there too?

**Comment by crazycs520** [comment]
LGTM

**Pull request merged**